test/gke/fake/cred.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
nit: maybe rename to credentials.js for readability?

Reply:
Renamed to `credentials.json`